### DISCUSSION TITLE
LATEX:"t" letter in text different form othr letters

### DISCUSSION THREAD

**Discussion by louwelyn-jeremy** [freeplane/freeplane#2042] [Help]
hi  I want to use a latex node to display a definition but all the "t" in my text are bold and in another font
the result
![image](https://github.com/user-attachments/assets/8123ff6f-7a99-4ae0-a98d-52f39839e44d)
the syntax
first try
```
\latex On appelle fonction polynome du second degré toutes fonctions $f$ définie sur $\mathbb{R}$ par une expression de la forme: \\ 
$f(x)=ax^2 + bx + c$ \\
avec $a,b$ et $c$ réels et a different de 0
``` 
second try:
```
\latex $\text{On appelle fonction polynome du second degré toutes fonctions}$ $f$ définie sur $\mathbb{R}$ par une expression de la forme: \\ 
$f(x)=ax^2 + bx + c$ \\
avec $a,b$ et $c$ réels et a different de 0
``` 
Where I'm wrong ?

**Comment by dpolivaev** [comment]
@fnatter ?

**Comment by fnatter** [comment]
hello,
you are not doing anything wrong, it is a problem either with JLaTeXMath or Swing or ...
I do not see this problem (1.11.11), also not with your latex input, but I think I have observed
similar issues in the past.
Cheers and Best Regards,
Felix

**Comment by louwelyn-jeremy** [comment]
I think I have the beginning of an explanation: it’s a problem of formatting.

![image](https://github.com/user-attachments/assets/5f264914-bc19-40f2-8cf9-f6d1cf59b8f5)

If I change the font size to 14, I have the problem. Even if I change the font… here i take all to 14

![image](https://github.com/user-attachments/assets/5a2b1c47-5861-4eb3-83b1-8185ffaa89cc)
